components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/logout/idpinit/exception/LogoutClientException.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change `Inc` to `LLC`?

Reply:
Since this file was created before the 2022, do we need to change `Inc` to `LLC`